src/main/resources/lang/default.lang

```diff
@@ -2579,6 +2579,11 @@ equipment slots:
 	off_hand: off hand slot, off hand
 	body: body slot, body armor slot, body armour slot
 
+# -- Villager Career Change Reasons
+villager career change reasons:
+	employed: employed, employment
+	losing_job: losing job, lost job
```

Comment:
I think the past tense ones are a bit confusing
in the example given (`event-career change reason is employed`) my initial reading is that the villager was previously employed and that's now changing, instead of them getting a job. It makes sense with a closer reading but I think limiting it to present tense is sensible and less confusing